src/Arpack.jl

```diff
@@ -310,6 +310,11 @@ julia> s.S
     that the size is smallest.
 """
 svds(A; kwargs...) = _svds(A; kwargs...)
+function _orth!(P)
+    Q,R = qr!(P)
+    rsign = [iszero(rval) ? one(rval) : sign(rval) for rval in diag(R)]
+    return Matrix(Q) * Diagonal(rsign)
```

Comment:
You can do `rmul!(Matrix(Q), Diagonal(rsign))` to avoid the extra allocation.

Reply:
Oh, right! Done. :)